config/config-default.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/ldflex-comunica/^2.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^1.0.0/components/context.jsonld"
+  ],
+  "@id": "urn:comunica:my",
+  "@type": "Runner",
+  "import": [
+    "files-cais:config/sets/http.json",
+    "files-cais:config/sets/join.json",
+    "files-cais:config/sets/rdf-dereference.json",
+    "files-cais:config/sets/rdf-parsers.json",
+    "files-cais:config/sets/context-preprocess-rdf-source-identifiers.json",
+    "files-cais:config/sets/resolve-federated.json",
```

Comment:
Federation is not really needed in LDflex, unless you have plans for this @RubenVerborgh?

Reply:
So weird thing: the original query-ldflex stopped working after I removed the federated resolver. Don't know why yet.

Reply:
Probably because we use `sources` (plural)?

Reply:
Ah yes, that's the reason. It _should_ work in the same way if you just use `source` though.

Reply:
@joachimvh Never mind my other comments for now then and just use `source`.